tsconfig.json

```diff
@@ -1,21 +1,32 @@
 {
   "compilerOptions": {
+    "allowJs": false,
+    "allowUnreachableCode": false,
+    "emitDecoratorMetadata": false,
+    "experimentalDecorators": true,
     "moduleResolution": "node",
-    "module": "es6",
+    "outDir": "lib/",
     "target": "es5",
-    "lib": [ "es2015", "dom" ],
-    "outDir": "lib",
+    "module": "esnext",
+    "lib": ["es2015", "dom"],
     "noEmitHelpers": true,
```

Comment:
We still have to do this, event with the new decorators?

Reply:
Have only been building for the purposes of running tests so haven't cleaned up this stuff. Also still need to do some research on best practices re: helpers for building typescript into intermediate libraries